packages/core/bootstrap/src/lib/server.ts

```diff
@@ -150,9 +150,13 @@ function initExpressMiddleware(app: express.Express) {
     windowMs,
     max, // limit each IP's requests per windowMs
     keyGenerator: () => '*', // use one key for all incoming requests
-    handler: ((_, res) => {
-      httpRateLimit.inc()
-      res.status(429).send('Too many requests, please try again later.')
+    handler: ((req, res) => {
+      if (req.url === '/health') {
+        res.status(200).send({ message: 'OK', version })
```

Comment:
I'd change this to `next()` just so we can have the option of using more complex healthchecks (I see in the code we already have a more complex redis check commented out)

Reply:
Seconded. It should just pass through.

Reply:
Good idea, just updated it